.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: julia
+julia:
+  - 1.3
+  - 1.4
```

Comment:
This isn't gonna work. Travis doesn't support rr.

Reply:
:(